.ci.yaml

```diff
@@ -321,7 +321,7 @@ targets:
 
   - name: Linux docs_test
     recipe: flutter/flutter
-    timeout: 60
+    timeout: 90 # https://github.com/flutter/flutter/issues/120901
```

Comment:
Should this have a `presubmit: false`?

Reply:
I don't think we call dashing to generate the docset on presubmit (the slow part), see https://ci.chromium.org/p/flutter/builders/try/Linux%20docs_test/23954?

So 60 should be more than enough.

Reply:
oh wait, this is for presubmit and postsubmit, eh?

Reply:
From the stdout console output, it looks like the pre- and post- submit do very similar work: https://www.diffchecker.com/4ePlpz7F/

However, the pre-submit seems to take ~10 minutes compared to post-submit ~60 minutes. Very weird.

Reply:
Checkout the timestamps from this bit of your diff:

```
Wed Feb 15 20:45:55 PST 2023: Zipping Flutter offline docs archive.
Wed Feb 15 20:55:39 PST 2023: Building Flutter docset.
Wed Feb 15 21:22:35 PST 2023: Moving offline data into place.
787M	doc/offline/flutter.docs.zip
637M	doc/offline/flutter.docset.tar.gz
```

Reply:
Gotcha. In that case increasing this timeout isn't concerning for pre-submit (but still necessary to prevent unnecessary post-submit flakes)

Reply:
Actually, I think the fix is to stop generating the docset post-submit in docs_test. we already generate them for docs_publish. The whole point of docs_test was to get some pre-submit signal BEFORE a change breaks dartdoc.

Reply:
but to unblock yourself, a 90 minute timeout SGTM, as long as we at least leave a TODO with at tracking issue

Reply:
> as long as we at least leave a TODO with at tracking issue

Yup the 90 minute timeout values link to this issue: https://github.com/flutter/flutter/issues/120901

I've updated that issue to track removing docset generation from post-submit `Linux docs_test`